[Term Entry] Go Math Functions Round() (#2706)
Codecademy/docs: content/go/concepts/math-functions/terms/round/round.md

---
Title: 'Round()'
Description: 'Rounds the given number to the nearest integer.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Math'
  - 'Functions'
CatalogContent:
  - 'learn-go'
  - 'paths/computer-science'
---

> The **`Round()`** function in Go is used to round a given number to the nearest integer. This function is part of the `math` package and can be imported to use in your Go programs.

Reviewer: ```suggestion
The **`Round()`** function in Go is used to round a given number to the nearest integer. This function is part of the `math` package which needs to be imported to use in Go programs.
```

## Syntax

```pseudo
result := math.Round(x)
```

Where:

> - `x`: is the number to be rounded

Reviewer: ```suggestion
- `x`: is the number to be rounded.
```
